optic.yml

```diff
@@ -0,0 +1,55 @@
+capture:
+  openapi.yml:
+    server:
+      # The command to run your server.
+      # Optional: If omitted, Optic assumes the server is running or started elsewhere.
+      # command: your-server-command
+      # The url where your server can be reached once running.
+      # Required: Can be overridden with '--server-override'.
+
+      # QUESTION: Is it ok that both this value and proxy_variable are the api gateway url?
+      url: https://rv42fyt9b0.execute-api.us-east-1.amazonaws.com/try-optic
```

Comment:
Since you're specifying this in `package.json` (`--server-override`) i think you might be able to omit this

Reply:
perfect